agent-bootstrap/src/main/java/io/opentelemetry/auto/typedspan/DelegatingSpan.java

```diff
@@ -91,6 +91,11 @@ public void setStatus(final Status status) {
     delegate.setStatus(status);
   }
 
+  @Override
+  public void recordException(Throwable throwable) {
+    delegate.recordException(throwable);
+  }
+
```

Comment:
ya, fix should be merged shortly in #824 👍

Reply:
rebased